pset/blinder.go

```diff
@@ -1,23 +1,25 @@
 package pset
 
 import (
-	"crypto/rand"
 	"errors"
 	"github.com/btcsuite/btcd/btcec"
 	"github.com/vulpemventures/go-elements/confidential"
 	"github.com/vulpemventures/go-elements/transaction"
 	"github.com/vulpemventures/go-secp256k1-zkp"
 )
 
+type randomNumberGenerator func(n int) ([]byte, error)
```

Comment:
we have 2 options here:
1. do not use any argument and then hardcode in our generateRandomNumber the `32` value
2. use an `interface{}` so the user is not forced to create his rng function with an `int` argument

I prefer number 1

Reply:
Please check now if this is ok now.